firebase-crashlytics/src/main/java/com/google/firebase/crashlytics/internal/common/CrashlyticsCore.java

```diff
@@ -337,7 +338,7 @@ public void logException(@NonNull Throwable throwable) {
    */
   public void log(final String msg) {
     final long timestamp = System.currentTimeMillis() - startTime;
-    commonWorker.submit(() -> controller.writeToLog(timestamp, msg));
+    diskWriteWorker.submit(() -> controller.writeToLog(timestamp, msg));
```

Comment:
send to diskWrite directly since there is nothing to processed with

Reply:
Will this maintain the consistency of user actions now that it's on the disk write worker?

For example,
diskWriteWorker.submit(a busy task that will take a second)
crashlytics.log("i am an important log")
throw new Exception()

Reply:
If you did `commonWorker.submitTask(diskWriteWorker.submit(...))` it will maintain order, and not use up a common thread while waiting for the disk write.